theme/open/views/imprint/imprint.hbs

```diff
@@ -16,40 +16,19 @@
 	<h2 class="h5">
-		<strong>Redaktion:</strong>
+		<strong>{{$t "imprint.imprint_rights.headline.editorialStuff" }}</strong>
 	</h2>
-	<p>Hasso Plattner Institut</p>
+	<p>{{$t "imprint.text.hassoPlattnerInstitute" }}</p>
 	<h2 class="h5">
-		<strong>Rechtliche Hinweise:</strong>
+		<strong>{{$t "imprint.imprint_platform_intro.headline.legalNotice" }}</strong>
 	</h2>
 	<p>
-		Die HPI Schul-Cloud ist ein nicht kommerzielles Pilotprojekt, welches vom HPI in Kooperation mit dem nationalen
-		Excellence-Schulnetzwerk MINT EC (<a href="https://www.mint-ec.de/" target="_blank"
-			rel="noopener">https://www.mint-ec.de/</a>) durchgeführt und durch das Bundesministerium für Bildung und Forschung
-		(BMBF) gefördert wird (mehr Informationen finden Sie unter:
-		<a href="https://hpi.de/schul-cloud" target="_blank" rel="noopener">https://hpi.de/schul-cloud</a>). Die Inhalte der
-		HPI Schul-Cloud stellen keine offiziellen Verlautbarungen des HPI dar. Für die Inhalte sind die jeweiligen
-		Autor/innen selbst verantwortlich. Ohne vorherige schriftliche Genehmigung ist eine kommerzielle Verbreitung der
-		über die HPI Schul-Cloud verfügbaren Inhalte ausdrücklich untersagt, außer diese sind ausdrücklich gekennzeichnet.
-		Die redaktionelle Verantwortung für den Inhalt verlinkter Webseiten anderer Anbieter (einschließlich der dort
-		verankerten Links zu Dritten), insbesondere die Verantwortung für die Beachtung der Bestimmungen des Jugendschutz-
-		und Datenschutzrechts, trägt in jeder Hinsicht der jeweilige Anbieter allein.</p>
-	<p>Das Hasso-Plattner-Institut (HPI) überprüft und aktualisiert regelmäßig die Inhalte seiner Websites. Trotz
-		größtmöglicher Sorgfalt können wir nicht ausschließen, dass einzelne Informationen auf unseren Seiten veraltet oder
-		nicht mehr zutreffend sind. Daher können wir keine Haftung für Aktualität, Genauigkeit und Vollständigkeit der
-		publizierten Informationen übernehmen. Das Gleiche gilt für alle Websites, die von den HPI-Seiten über Hyperlinks
-		erreichbar sind. Für alle diese Links gilt: Wir erklären ausdrücklich, dass wir keinen Einfluss auf die Gestaltung
-		und die Inhalte der verlinkten Seite haben. Deshalb distanzieren wir uns hiermit von allen Inhalten aller verlinkten
-		Seiten und machen uns ihre Inhalte nicht zu Eigen. Diese Erklärung gilt für alle auf unserer Seite angebrachten
-		Links zu fremden Seiten. Für den Inhalt dieser verlinkten Seiten ist das HPI nicht verantwortlich. Weiterhin behält
-		sich das HPI jederzeit Änderungen und Ergänzungen der angezeigten Informationen vor. Inhalt und Struktur der
-		HPI-Website sind durch das Copyright geschützt. Die Weiterverwendung von Informationen und Daten ist nur nach
-		unserer Zustimmung zulässig. Wir wünschen Ihnen einen informativen Aufenthalt auf unseren Websites und freuen uns
-		über Vorschläge, Wünsche oder Kritik zur HPI Schul-Cloud.</p>
+		{{$t "theme.open.imprint.text.imprintLegalNoticeOpen"}}</p>
```

Comment:
use {{{ }}} and `<p>
{{$t "theme.open.imprint.text.imprintLegalNoticeOpen"}}</p>` should be in the same line